src/cljs/owlet_ui/components/branch.cljs

```diff
@@ -4,6 +4,20 @@
             [reagent.core :as reagent :refer [atom]]
             [camel-snake-kebab.core :refer [->kebab-case]]))
 
+(def hover-image (reagent/atom ""))
+
+(defn get-random [images]
+  (get images (rand-int (count images))))
+
+(defn set-hover [images]
+  (let [image (get-random images)]
+    (if (> (count images) 1)
+      ; prevent the same image twice in a row
+      (if (== image @hover-image)
+        (set-hover images)
```

Comment:
@tylerperkins there are cases when local component state makes more sense. I think this is one. If the state management for this `component/view` was applicable to another component or another part of the app; I would say yes 💯 let's make a subscription with a dispatcher and all but I don't this it applies here. It would just complicate things, imo.